app/code/Magento/CatalogImportExport/Model/Import/Product/Type/AbstractType.php

```diff
@@ -533,11 +533,6 @@ public function prepareAttributesWithDefaultValueForSave(array $rowData, $withDe
      */
     public function clearEmptyData(array $rowData)
     {
-        foreach ($this->_getProductAttributes($rowData) as $attrCode => $attrParams) {
```

Comment:
@RomaKis I would consider that we could now deprecate this method now if it is not doing anything and also remove the usage of this method from any place we can find.